packages/web-components/src/components/structured-list/structured-list-body.ts

```diff
@@ -0,0 +1,17 @@
+import BXStructuredListBody from 'carbon-web-components/es/components/structured-list/structured-list-body';
+import { customElement } from 'lit-element';
+import ddsSettings from '../../internal/vendor/@carbon/ibmdotcom-utilities/utilities/settings/settings';
+import styles from './structured-list.scss';
+
+const { stablePrefix: ddsPrefix } = ddsSettings;
+
+@customElement(`${ddsPrefix}-structured-list-body`)
+class DDSStructuredListBody extends BXStructuredListBody {
```

Comment:
Just thinking about some of the formatting options that Content Authors might be adding to the table cell body. @oliviaflory or @jwitkowski79 Do you know if we need to account for paragraph style like bullets, unordered list, link, etc in these table cells? If so, does it makes sense for us to extend from `dds-content-block-paragraph` (see - https://carbon-design-system.github.io/carbon-for-ibm-dotcom/canary/web-components/?path=/story/components-content-block-simple--with-link-list)?